### ISSUE TITLE
Update Ruby Support; Drop 2.8, 3.0; Add 3.2, 3.3

### ISSUE THREAD

**Issue by zacheryph** [patbenatar/rbexy#97]
Wanted to look at dropping support for Ruby 2.8 (EoL almost a year ago) and 3.0 (expected EoL end of March) and add support for Ruby 3.2, 3.3.

Adding 3.2 does not affect the test suite at all.

There are backtrace related tests affected on Ruby 3.3. A group are easy to fix (`(eval)` was changed to `(eval at filename:lineno)`.) In the other the backtrace is _hiding_ the source of a method block within templates that fail. I need to look into this a little more (I didn't see anything regarding these backtrace changes at a quick glance on the Ruby release notes.)

**Comment by patbenatar** [comment]
Agreed on all. We currently run on Ruby 3.2 so I know it works there. If you can make it work on 3.3 that'd be great!
